app/src/main/java/com/yellowmessenger/ymchatexample/MainActivity.java

```diff
@@ -65,6 +65,8 @@ protected void onCreate(Bundle savedInstanceState) {
         // This is an optional parameter
         ymChat.config.customLoaderUrl = "https://yellow.ai/images/Logo.svg";
 
+       // ymChat.config.customBaseUrl = "https://staging.yellowmessenger.com";
```

Comment:
Also remove this debugging code for staging

Reply:
sure